### DISCUSSION TITLE
Previewer with Blazor Server app

### DISCUSSION THREAD

**Discussion by paulpitchford** [QuestPDF/QuestPDF#533] [Q&A] [answered]
Hi,

I have a solution that contains a project for my web app, a project for my data access and I've created a project for my reports using QuestPDF.

My page in the blazor app is creating the report like so:

```
if (order != null)
{
    DeliveryNote document = new DeliveryNote(order);

    document.ShowInPreviewer();
}
```

However, when the web app runs, the report doesn't get picked up by the previewer so I cannot see it in the previewer. I use the same code in a console app and it loads fine.

Is there a way to preview the report in my working environment?

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Since this discussion was opened, the Previewer has been replaced by the Companion app (available since version 2024.10.0 via `ShowInCompanion()`), which offers an improved experience across different project types.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
